### DISCUSSION TITLE
Hindi Stopwords Removeing

### DISCUSSION THREAD

**Discussion by munishmangla98** [explosion/spaCy#12760] [Help: Best practices]
Hello to all,
i am working on a sentiment analysis project. I want to remove the stop words of **hindi language**,kindly let me know what API should i have to use to remove the stopwords of hindi.
****

**Comment by danieldk** [comment]
You could refer to this earlier discussion: https://github.com/explosion/spaCy/discussions/10942#discussioncomment-2919121

This should also work for loaded models. For instance:

```
>>> import spacy
>>> nlp = spacy.load('en_core_web_lg')
>>> [token.is_stop for token in nlp("Is this a stop word?")]
[True, True, True, False, False, False]
>>> from spacy.lang.en import EnglishDefaults
>>> EnglishDefaults.stop_words = set()
>>> nlp = spacy.load('en_core_web_lg')
>>> [token.is_stop for token in nlp("Is this a stop word?")]
[False, False, False, False, False, False]
```
